csharp/device/Microsoft.Azure.Devices.Client/DeviceClientHelper.cs

```diff
@@ -209,7 +351,14 @@ public Task<Message> ReceiveAsync()
             await this.EnsureOpenedAsync(false);
             await this.OnSendEventAsync(messages);
         }
+#endif
 
+#if NETMF
+        protected void EnsureOpened(bool explicitOpen)
+        {
+
```

Comment:
just checking if this should be empty? maybe if it's useless we can remove it altogether?